src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Rate/Rate.ts

```diff
@@ -318,6 +321,7 @@ export var directiveFactory = (template : string, adapter : IRateAdapter<RIRateV
                             })
                             .then(() => undefined);
                     }).finally<void>(() => {
+                        scope.isCast = true;
```

Comment:
I'm not sure about that. I have seen this many times implemented as "click to see the results" I think seeing the results can effect how you vote. m-show-results-cast seems like additional functionality + I'm not sure we even need m-show-results-click !